### PR TITLE
PERFSCALE-457, updated routing optimization docs

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://jira.coreos.com/browse/PERFSCALE-457

### CODE REVIEW

File: modules/router-performance-optimizations.adoc [hunk 1]
```diff
@@ -1,70 +1,10 @@
...
-reload intervals, can cause instantiation of many HAProxy processes.
-These processes must handle old connections, which were started
-before the HAProxy configuration reload. A large number of these processes is
-undesirable, as it will exert unnecessary load on the system and can
-lead to issues, such as out of memory conditions.
-
-Router environment variables affecting this
-behavior are `ROUTER_DEFAULT_TUNNEL_TIMEOUT`, `ROUTER_DEFAULT_CLIENT_TIMEOUT`,
-`ROUTER_DEFAULT_SERVER_TIMEOUT`, and `RELOAD_INTERVAL` in particular.
+You can modify the deployment, but if the Operator is enabled, the configuration
```
Comment (ahardin-rh) [inline modules/router-performance-optimizations.adoc]: @jmencak What is the Operator's official name? Trying to get a cluster up to check myself, but it keeps failing.

Comment (jmencak) [reply modules/router-performance-optimizations.adoc]: The ingress operator?

https://github.com/openshift/cluster-ingress-operator 

```
$ oc get pods -n openshift-ingress-operator
NAME                                READY   STATUS    RESTARTS   AGE
ingress-operator-7c5fd8bbfb-x74dm   1/1     Running   0          7h39m
```

File: modules/router-performance-optimizations.adoc [hunk 2]
```diff
@@ -1,70 +1,10 @@
 [id="router-performance-optimizations_{context}"]
 = Router performance optimizations
 
-*Setting the maximum number of connections*
+{product-title} no longer supports modifying router deployments by setting
+environment variables such as  ROUTER_THREADS, `ROUTER_DEFAULT_TUNNEL_TIMEOUT`,
```
Comment (jmencak) [inline modules/router-performance-optimizations.adoc]: backticks missing for ROUTER_THREADS?

**Comment by ahardin-rh** [comment]
@jmencak Thanks for your feedback. Updated!

@mffiedler Can you please provide QE review?

File: modules/router-performance-optimizations.adoc [hunk 3]
```diff
@@ -1,70 +1,10 @@
...
-reload intervals, can cause instantiation of many HAProxy processes.
-These processes must handle old connections, which were started
-before the HAProxy configuration reload. A large number of these processes is
-undesirable, as it will exert unnecessary load on the system and can
-lead to issues, such as out of memory conditions.
-
-Router environment variables affecting this
-behavior are `ROUTER_DEFAULT_TUNNEL_TIMEOUT`, `ROUTER_DEFAULT_CLIENT_TIMEOUT`,
-`ROUTER_DEFAULT_SERVER_TIMEOUT`, and `RELOAD_INTERVAL` in particular.
+You can modify the deployment, but if the Ingress Operator is enabled, the
```
Comment (jmencak) [inline modules/router-performance-optimizations.adoc]: Perhaps be more specific and say ... modify the *router* deployment, but ... ?

Comment (ahardin-rh) [reply modules/router-performance-optimizations.adoc]: Updated. Thanks!

**Comment by mffiedler** [comment]
@schituku Please review for QE

**Comment by ahardin-rh** [comment]
@schituku Updated preview build: http://file.rdu.redhat.com/~ahardin/10012019/routing-opt-updates/scalability_and_performance/routing-optimization.html#router-performance-optimizations_routing-optimization

**Review by schituku — APPROVED** [review]
/lgtm

**Pull request merged**